tests/public_api/test_connector.py

```diff
@@ -133,6 +133,7 @@ def test_post_models_mocked(
     assert payload == expected_payload
 
 
+@pytest.mark.skip
```

Comment:
```suggestion
@pytest.mark.skip(reason="lacking ssl certificate")
```

Reply:
added a more general reason here 36ebc12
(also for test_extract_mex_person_items)